specs/linux_file_system_hierarchy.md

```diff
@@ -184,11 +184,24 @@ these should be placed in a subdirectory of `/usr/lib/` instead.
 Static, private vendor data that is compatible with all architectures
 (though not necessarily architecture-independent).
 Note that this includes
-internal executables or other binaries that are not regularly invoked from a shell.
+internal binaries or other programs that are not regularly invoked from a shell.
 Such binaries may be for any architecture supported by the system.
 Do not place public libraries in this directory,
 use `$libdir` (see below), instead.
 
+### `/usr/libexec/`
+
+Vendor binaries or other programs that are not regularly invoked from a shell.
+Such binaries may be for any architecture supported by the system.
+
+Packages may either place such programs directly in `/usr/libexec/`,
+in a subdirectory of `/usr/libexec/` named after the package,
+or in some subdirectory of `/usr/lib/`.
+Using `/usr/libexec/` is better if the package only has a single such file,
+and a subdirectory under `/usr/lib/` is better if the package
+has other non-executable files and needs to have directory there anyway.
+This document does not mandate any specific choice.
```

Comment:
this is BS, please remove this.

When i wrote this I expressly said a the top that the thing is a skeleton only, and people can make extensions. But here you are adding something that has no reason to exist, is not generally implemented, and fully redundant.

Let's please kill this again, and cover it via the "skeleton" thing.

I am fine if you mention libexec as an example for such extensions covered by the skeleton clause, but let's *not* suggest to people to use libexec, because it's pointless.